### PR TITLE
Install glide to manage wum-client dependancies

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
Install the glide to manage the dependencies in wum-client.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -147,6 +147,13 @@ RUN wget -P /build/software/go https://dl.google.com/go/go1.10.linux-amd64.tar.g
 RUN wget -P /build/software/go https://dl.google.com/go/go1.12.5.linux-amd64.tar.gz \
     && tar -xzf /build/software/go/go1.12.5.linux-amd64.tar.gz --directory /build/software/go && mv /build/software/go/go /build/software/go/go-1.12.5 && rm /build/software/go/go1.12.5.linux-amd64.tar.gz
 
+RUN cp -r /build/software/go/go-1.12.5 /usr/local/go
+
+ENV GOPATH /go
```
Comment (kasunbg) [inline Dockerfile]: Every path used for building needs to be located under /build.

File: Dockerfile [hunk 2]
```diff
@@ -147,6 +147,13 @@ RUN wget -P /build/software/go https://dl.google.com/go/go1.10.linux-amd64.tar.g
 RUN wget -P /build/software/go https://dl.google.com/go/go1.12.5.linux-amd64.tar.gz \
     && tar -xzf /build/software/go/go1.12.5.linux-amd64.tar.gz --directory /build/software/go && mv /build/software/go/go /build/software/go/go-1.12.5 && rm /build/software/go/go1.12.5.linux-amd64.tar.gz
 
+RUN cp -r /build/software/go/go-1.12.5 /usr/local/go
```
Comment (kasunbg) [inline Dockerfile]: Shall we keep everything under /build mount? You can symlink to a versionless folder like /build/software/go/go?

File: Dockerfile (hunk 1 above)
Comment (KavinduZoysa) [reply Dockerfile]: Change the path to /build/software/go/go

File: Dockerfile (hunk 2 above)
Comment (KavinduZoysa) [reply Dockerfile]: Change the path to /build/software/go/go

**Review by kasunbg — APPROVED** [review]

**Pull request merged**